Source/Dafny/Compilers/Compiler-java.cs

```diff
@@ -2195,6 +2196,19 @@ private class GenericArrayElementLvalue : ILvalue {
       }
     }
 
+    private void EmitRuntimeJar(string targetDirectory) {
```

Comment:
This is definitely "weird", but not a lot weirder than the old way. :) 

It's at least more consistent with the behavior of the other languages.